rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/NodeSourceWindow.java

```diff
@@ -969,6 +974,12 @@ public void setNormalNodeSourceWindowLabel() {
         this.nodeSourceWindowLabel.setContents(WINDOW_HEADER);
     }
 
+    public void changeToAdvancedConfiguration() {
+        isAdvanced.setDefaultValue(true);
+        isAdvanceChangedHandler();
+
```

Comment:
I think you can remote this jump to next line

Reply:
ok thanks